django_codemod/visitors/core.py

```diff
@@ -0,0 +1,11 @@
+from django_codemod.constants import DJANGO_110, DJANGO_20
+from django_codemod.visitors.base import BaseSimpleModuleRenameTransformer
+
+
+class URLResolversTransformer(BaseSimpleModuleRenameTransformer):
```

Comment:
Could we have a couple tests for this please? Just to confirm the `rename_from` and `rename_to` are properly configured

Reply:
Sure, and yup for docs also.

Reply:
Done (one test, sorry, and sorry for delays)